compiler/rustc_smir/src/rustc_smir/mod.rs

```diff
@@ -1497,30 +1497,32 @@ impl<'tcx> Stable<'tcx> for ty::ClauseKind<'tcx> {
     type T = stable_mir::ty::ClauseKind;
 
     fn stable(&self, tables: &mut Tables<'tcx>) -> Self::T {
-        use ty::ClauseKind::*;
```

Comment:
SMIR: this glob import doesn't work anymore because we don't import variants when glob-importing from a type alias. I opted to just refer to these variants by their qualified name -- imo, it's also not that big of a deal, since we're literally doing the same thing for `PredicateKind` in this file.